binaries/platform/platform.go

```diff
@@ -44,6 +44,17 @@ func Name() string {
 	return runtime.GOOS
 }
 
+func Arch() string {
+	// should be x86 or arm64
+	if runtime.GOARCH == "amd64" {
+		return "x64"
+	} else if runtime.GOARCH == "arm64" {
+		return "arm64"
+	} else {
+		panic(fmt.Errorf("unsupported arch: %s", runtime.GOARCH))
+	}
```

Comment:
I think let's:
- use a switch/case as it is a bit more elegant
- default to x64 and just log a warning `log.Printf("warning: unsupported architecture %s, falling back to x64", runtime.GOARCH)`

because I think it could happen that the value is not right but the x64 still might work